PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/lib/legacy/function/FunctionCoreToolsCombobox.java

```diff
@@ -94,6 +99,12 @@ public FunctionCoreToolsCombobox(Project project, boolean includeSettings) {
         }
     }
 
+    private void openAzureSettingsPanel() {
+        final Action<Void> openSettingsAction = AzureActionManager.getInstance().getAction(ResourceCommonActionsContributor.OPEN_AZURE_SETTINGS);
```

Comment:
add todo to await action

Reply:
We need to refactor the action framework for execute sync, add work item to track it